docs/proposals/compute-resource-metrics-api.md

```diff
@@ -92,39 +104,17 @@ only be available through the kubelet. The types of metrics are detailed
 `/apis/metrics/v1alpha1/`
 
 - `/` - discovery endpoint; type resource list
-- `/rawNodes` - raw host metrics; type `[]metrics.RawNode`
-  - `/rawNodes/localhost` - The only node provided is `localhost`; type
```

Comment:
I think you should still call out the localhost node as the only one which will be served by the kubelet.